src/App.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import Header from "./components/Header";
+import CreateArea from "./components/CreateAreaObject";
+import Note from "./components/Note";
+
+import "./App.css";
+
+function App() {
+  const [notes, setNotes] = useState([]);
+
+  function addNote(newNote) {
+    setNotes((prevNotes) => [...prevNotes, newNote]);
+  }
+
+  function deleteNote(id) {
+    setNotes((prevNotes) => prevNotes.filter((note) => note.id !== id));
+  }
+
+  function handleEdit(id, updateNote, handleEdits) {
```

Comment:
this should take one argument: note

note should have this interface

{
  id,
  content,
  title
}